cider/function/FunctionLookupEngine.h

```diff
@@ -34,8 +34,33 @@
 #include "FunctionSignature.h"
 #include "Type.h"
 #include "function/SubstraitFunctionCiderMappings.h"
+#include "include/cider/CiderSupportPlatType.h"
 
-enum PlatformType { SubstraitPlatform, PrestoPlatform, SparkPlatform };
+namespace io::substrait {
+
+class VeloxFunctionMappings : public FunctionMapping {
```

Comment:
Hmm, it's not velox but either Spark or Presto, right? Velox itself didn't bring definition of function/operator semantics.

Reply:
Done. Rename to presto.